<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtRateExamplesDialogNewName</class>
 <widget class="QDialog" name="QtRateExamplesDialogNewName">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>497</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Evalueer voorbeelden</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label_concept_name">
     <property name="text">
      <string>Concept: orde houden</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_above_list">
     <property name="text">
      <string>Voorbeelden/toelichting:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QListWidget" name="list">
     <property name="verticalScrollBarPolicy">
      <enum>Qt::ScrollBarAsNeeded</enum>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="resizeMode">
      <enum>QListView::Adjust</enum>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <item>
      <property name="text">
       <string>A</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicRed14x14.png</normaloff>:/images/PicRed14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>B</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicOrange14x14.png</normaloff>:/images/PicOrange14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>C</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicYellow14x14.png</normaloff>:/images/PicYellow14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>D</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicGreen14x14.png</normaloff>:/images/PicGreen14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>E</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicCyan14x14.png</normaloff>:/images/PicCyan14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>F</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicBlue14x14.png</normaloff>:/images/PicBlue14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>G</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicPurple14x14.png</normaloff>:/images/PicPurple14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>H</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicWhite14x14.png</normaloff>:/images/PicWhite14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>I</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicWhite14x14.png</normaloff>:/images/PicWhite14x14.png</iconset>
      </property>
     </item>
     <item>
      <property name="text">
       <string>J</string>
      </property>
      <property name="icon">
       <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
        <normaloff>:/images/PicWhite14x14.png</normaloff>:/images/PicWhite14x14.png</iconset>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="title">
      <string>Markeertekst</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QPushButton" name="button_prof">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Beroepsdomein</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicRed14x14.png</normaloff>:/images/PicRed14x14.png</iconset>
        </property>
        <property name="checkable">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_organisations">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Organisaties</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicOrange14x14.png</normaloff>:/images/PicOrange14x14.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_social">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Sociale omgeving</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicYellow14x14.png</normaloff>:/images/PicYellow14x14.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_target_audience">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Doelgroep</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicGreen14x14.png</normaloff>:/images/PicGreen14x14.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_ti_knowledge">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Technisch instrumentele aspecten</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicCyan14x14.png</normaloff>:/images/PicCyan14x14.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_prof_development">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Professionele ontwikkeling</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicBlue14x14.png</normaloff>:/images/PicBlue14x14.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_misc">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="text">
         <string>&amp;Overig</string>
        </property>
        <property name="icon">
         <iconset resource="../../Projects/ProjectBrainweaver/Brainweaver.qrc">
          <normaloff>:/images/PicPurple14x14.png</normaloff>:/images/PicPurple14x14.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="button_ok">
        <property name="text">
         <string>&amp;OK</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../Projects/ProjectBrainweaver/Brainweaver.qrc"/>
 </resources>
 <connections/>
</ui>
